src/sentry/integrations/github/webhook.py

```diff
@@ -110,11 +110,20 @@ def __call__(self, event: Mapping[str, Any], host: str | None = None) -> None:
 
                     if features.has("organizations:integrations-auto-repo-linking", org):
                         try:
-                            provider.create_repository(repo_config=config, organization=rpc_org)
+                            _, repo = provider.create_repository(
```

Comment:
Are you naming the first value here `_` since we don't use it? Is that a convention or anything (just curious)

Reply:
yes